moto/route53/models.py

```diff
@@ -64,10 +64,26 @@ def __init__(self, health_check_id, caller_reference, health_check_args):
         self.measure_latency = health_check_args.get("measure_latency") or False
         self.inverted = health_check_args.get("inverted") or False
         self.disabled = health_check_args.get("disabled") or False
-        self.enable_sni = health_check_args.get("enable_sni") or False
-        self.children = health_check_args.get("children") or None
+        self.enable_sni = health_check_args.get("enable_sni") or True
         self.caller_reference = caller_reference
 
+    def add_children(self, children):
+        if children and isinstance(children, list):
+            self.children = children
+        elif children and isinstance(children, str):
+            self.children = [children]
+        else:
+            self.children = None
```

Comment:
Is this desired behaviour?
When calling `update_healthcheck` without the `children`-argument, this removes any existing children. It seems more intuitive to keep the existing children, if the user doesn't explicitly say they want to change/update them.

Reply:
Correct. if calling `update_healthcheck` without `--child-health-checks` argument - its not removing any children. Fixed this.

But it will overwrite all children if `--child-health-checks` argument is present with valid children.